engine/rpcserver.go

```diff
@@ -1658,6 +1659,16 @@ func (s *RPCServer) GetHistoricCandles(_ context.Context, req *gctrpc.GetHistori
 			Volume: candles.Candles[i].Volume,
 		})
```

Comment:
Further to this, this one is actually in scope. You can misattribute asset type:
```
.\gctcli.exe gethistoriccandlesextended -e=Bitstamp --pair="LTC-USD" -asset=index -sync=true -i=86400
```

Bitstamp has no support for index. And now we have index data in the database.

I did the same with Huobi & Kraken for margin. I can assume this impacts all exchanges

Reply:
klineIntervalEnabled() is no longer exported and has been replaced with a ValidateKline() method

```go
func (e *Base) ValidateKline(pair currency.Pair, a asset.Item, interval kline.Interval) error {}
```

That will check if asset is enabled pair is enabled and interval is supported by the exchange